From a2eae8c6162ff71c1ee68dfd8338078f8ce00798 Mon Sep 17 00:00:00 2001
From: Nishanth Menon <nm@ti.com>
Date: Sat, 5 Dec 2020 09:17:30 -0600
Subject: [PATCH 1/8] tcl: discover

Add a simple discovery tcl

openocd -f discover.cfg should get us started on new SoCs for figuring
out what specific tapID is being used etc...

New devices http://openocd.org/doc/html/TAP-Declaration.html

Change-Id: I7c89f124740247749cb06210a9b2d96c32e30dc6
Signed-off-by: Nishanth Menon <nm@ti.com>
---
 tcl/discover.cfg | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 tcl/discover.cfg

diff --git a/tcl/discover.cfg b/tcl/discover.cfg
new file mode 100644
index 000000000000..c5176684072a
--- /dev/null
+++ b/tcl/discover.cfg
@@ -0,0 +1,12 @@
+
+# J721E EVM has an xds110 onboard.
+source [find interface/xds110.cfg]
+
+transport select jtag
+
+# default JTAG configuration has only SRST and no TRST
+reset_config srst_only srst_push_pull
+
+# delay after SRST goes inactive
+adapter srst delay 20
+adapter speed 200
-- 
2.29.2

